tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -126,13 +123,23 @@ GRACEFUL_EXIT       If set, and if the DISTRO configuration specifies a
                     specific distributions.
                     Default value: <not set>
 
+IMAGE_REGISTRY      Hostname for the image registry used to pull down the rootfs
+                    build image.
+                    Default value: docker.io
+
 KERNEL_MODULES_DIR  Path to a directory containing kernel modules to include in
                     the rootfs.
                     Default value: <empty>
 
 ROOTFS_DIR          Path to the directory that is populated with the rootfs.
                     Default value: <${script_name} path>/rootfs-<DISTRO-name>
 
+RUST_AGENT          When set to "no", build kata-agent from go agent instead of kata-rust-agent
+                    Default value: "yes"
+
+RUST_AGENT_PKG      URL of the Git repository hosting the agent package.
+                    Default value: ${RUST_AGENT_PKG}
+
```

Comment:
I'd put those one if the DOCKER_RUNTIME re-shuffle as part of a different patch in, but in the same series.